corehq/apps/app_manager/templates/app_manager/module_view.html

```diff
@@ -272,7 +272,7 @@
                 <div class="form-group">
                     <label class="col-sm-2 control-label" for="comment-id">{% trans "Comment" %}</label>
                     <div class="col-sm-4">
-                        <textarea id="comment-id" name="comment" class="form-control">{{ module.comment }}</textarea>
+                        <inline-edit params="name: 'comment', id:'comment-id', value: '{{ module.comment|escapejs }}'"></inline-edit>
```

Comment:
For custom elements, yes. The other option is to use the `component` bind and pass it a name and a hash of parameters (see http://knockoutjs.com/documentation/component-binding.html). I like the custom element a little better but don't feel that strongly about it.